single.php

```diff
@@ -9,20 +9,62 @@
  * @since    Timber 0.1
  */
 
+// Add custom css class for body element
+function add_body_classes_for_post( $classes ) {
+	$classes[] = 'post_body';
+	return $classes;
+}
+add_filter( 'body_class', 'add_body_classes_for_post' );
+
+
+// Initializing variables.
+$context                     = Timber::get_context();
+$post                        = Timber::query_post();
+$context['post']             = $post;
 
-$context = Timber::get_context();
-$post    = Timber::query_post();
 
-$page_meta_data             = get_post_meta( $post->ID );
-$articles_title             = $page_meta_data['p4_articles_title'][0];
-$articles_count             = intval( $page_meta_data['p4_articles_count'][0] );
-$articles_count             = $articles_count == 0 ? 3 : $articles_count;
-$context['author_override'] = $page_meta_data['p4_author_override'][0];
+// Get the cmb2 custom fields data
+// Articles block parameters to populate the articles block
+// Author override parameter. If this is set then the author profile section will not be displayed
+$page_meta_data              = get_post_meta( $post->ID );
+$articles_title              = $page_meta_data['p4_articles_title'][0];
+$articles_count              = intval( $page_meta_data['p4_articles_count'][0] );
+$articles_count              = $articles_count == 0 ? 3 : $articles_count;
```

Comment:
Use YODA condition here to comply with standards and do type checking at the same time by adding ===
`0 === $articles_count`